plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -7780,3 +7780,20 @@ table.bar_chart {
 		}
 	}
 }
+
+#postdivrich.woocommerce-product-description {
+	margin-bottom: 0px;
+	
+	.wp-editor-tools {
+		background: none;
+		padding-top: 0px;
+		width: 100%;
+	}
+	.wp-editor-wrap {
+		margin: 6px 12px 0;
+	}
+	#post-status-info {
+		margin: 0px 12px 12px;
```

Comment:
Do we have access to our usual scss vars in here?  If so we could use gap vars like `$gap-small`.

Reply:
Unfortunately it doesn't seem like we do.